Vizzuality/heco-invest
# feat: API endpoint for updating project developer

New API endpoint which allows to update Project Developer information.

![Screenshot 2022-03-29 at 11 06 59](https://user-images.githubusercontent.com/20660167/160577715-9d8aaaf2-6ac7-4a66-b642-b4d8ddc318d6.png)

## Testing instructions

Open Rswag documentation and try to update project developer of user which is already project developer. Don't forget to check out `Send empty value` for fields which you don't wanna update ;).

## Tracking

https://vizzuality.atlassian.net/browse/LET-232
